app/javascript/readingList/readingList.jsx

```diff
@@ -88,24 +88,22 @@ export class ReadingList extends Component {
   toggleArchiveStatus = (event, item) => {
     event.preventDefault();
 
-    const { statusView, items, totalCount } = this.state;
```

Comment:
This state property is not required as the total count can be derived from existing state via `items.length`.